libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -3363,16 +3375,25 @@ void IRGeneratorForStatements::revertWithError(
 	vector<ASTPointer<Expression const>> const& _errorArguments
 )
 {
+	bool needsAllocation = true;
+	if (auto size = staticEncodingSize(_parameterTypes))
```

Comment:
```suggestion
	if (optional<size_t> size = staticEncodingSize(_parameterTypes))
```
The type is not mentioned anywhere in the statement (at least on the right hand side), so it would be nice to make it explicit in this case.

Reply:
Is that really so :-)?